src/server/course/course.service.ts

```diff
@@ -73,8 +73,7 @@ export class CourseService {
       .addSelect('c."termPattern"', 'termPattern')
       .addSelect('c."isUndergraduate"', 'isUndergraduate')
       .addSelect('c."isSEAS"', 'isSEAS')
-      .addSelect('c."sameAsId"', 'sameAsId')
-      .addSelect("CONCAT_WS(' ', p.prefix, p.number)", 'sameAs')
```

Comment:
Since we're dropping this line we no longer need `"p"` and can also get rid of line `81`, where we `leftJoinAndSelect` on the `sameAsId`

Reply:
Good point! Fixed in bce2ff7